js/components/volumeControls.js

```diff
@@ -92,6 +92,9 @@ var VolumeControls = createReactClass({
    * @returns {String} The current volume in a screen reader friendly format (i.e. 20% volume).
    */
   getAriaValueText: function() {
+    if (this.props.controller.state.volumeState.muted) {
```

Comment:
Can we add a unit test that checks that the correct label is returned?

Reply:
done